nautobot/circuits/filters.py

```diff
@@ -145,7 +147,8 @@ class CircuitFilterSet(NautobotFilterSet, StatusModelFilterSetMixin, TenancyMode
     provider_network = NaturalKeyOrPKMultipleChoiceFilter(
         field_name="circuit_terminations__provider_network",
         queryset=ProviderNetwork.objects.all(),
-        label="Provider Network (slug or ID)",
+        to_field_name="name",
```

Comment:
Now that `name` is the default for `NaturalKeyOrPKMultipleChoiceFilter.to_field_name` should these lines be removed?

Reply:
I'm open to discussion on this point. I think there's a certain value in explicit-over-implicit, but you're right that it is *technically* redundant.

Reply:
Per team discussion we'll keep these as-is.